src/web/client/common/portalSchemaReader.ts

```diff
@@ -0,0 +1,32 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { portal_schema_data } from "./portalSchema";
+
+export function readSchema() {
```

Comment:
why do we need this function? The only thing it is doing is invoking another function.